libs/langgraph/langgraph/checkpoint/base.py

```diff
@@ -203,6 +213,14 @@ async def aput(
     ) -> RunnableConfig:
         raise NotImplementedError
 
+    async def aput_writes(
+        self,
+        config: RunnableConfig,
+        writes: List[Tuple[str, Any]],
+        task_id: str,
+    ) -> None:
+        raise NotImplementedError
```

Comment:
Should we default this to running put_writes in executor or does that mess up the error?

Reply:
No, let's make consistent with the other methods